src/test/java/org/fundacionjala/coding/dennis/BankOcrTest.java

```diff
@@ -0,0 +1,135 @@
+package org.fundacionjala.coding.dennis;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ *
+ */
+public class BankOcrTest {
+
+    /**
+     *
+     */
+    @Test
+    public void testNormal1() {
+        String line1 = "    _  _     _  _  _  _  _  _ ";
+        String line2 = "  | _| _||_||_ |_   ||_||_|| |";
+        String line3 = "  ||_  _|  | _||_|  ||_| _||_|";
+        String exp = "1234567890";
+        assertEquals(exp, BankOcr.decode(line1, line2, line3));
+    }
+
+    /**
+     *
+     */
+    @Test
+    public void testNormal2() {
+        String line1 = " _  _  _  _  _  _  _  _  _  _ ";
+        String line2 = "| || || || || || || || || || |";
+        String line3 = "|_||_||_||_||_||_||_||_||_||_|";
+        String exp = "0000000000";
+        assertEquals(exp, BankOcr.decode(line1, line2, line3));
+    }
+
+    /**
+     *
+     */
+    @Test
+    public void testNormal3() {
+        String line1 = "       _  _  _  _  _  _     _ ";
+        String line2 = "  ||_|  | _||_||_  _| _|  ||_|";
+        String line3 = "  |  |  ||_ |_| _||_  _|  ||_|";
+        String exp = "1472852318";
+        assertEquals(exp, BankOcr.decode(line1, line2, line3));
+    }
+
+    /**
+     *
+     */
+    @Test
+    public void testChecksumRight() {
+        String line1 = "345882865";
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "345882865" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)